AntistasiOfficial.Altis/Missions/SUP_CitySupply.sqf

```diff
@@ -0,0 +1,246 @@
+if (!isServer and hasInterface) exitWith {};
+
+params ["_marker", "_type"];
+//[3,[],[],[],[],[]] params ["_countBuildings","_targetBuildings","_allGroups","_allSoldiers","_allVehicles","_leafletCrates"];
+private ["_targetPosition","_targetName","_duration","_endTime","_task","_spawnPosition","_missionVehicle","_crate","_range","_allBuildings","_usableBuildings","_index","_perimeterBuildings","_currentBuilding","_lastBuilding","_bPositions","_groupType","_params","_group","_dog","_leaflets","_drop"];
+
+
+_tskTitle = "City Supplies";
+_tskDesc = "City Supplies Description";
+_tskDesc_fail = "You failed delivering a paket, very impressive";
+_tskDesc_success = "You crazy little postman";
+
+_targetPosition = getMarkerPos _marker;
+_targetName = [_marker] call AS_fnc_localizar;
+_range = [_marker] call sizeMarker;
+
+_allBuildings = nearestObjects [_targetPosition, ["Building"], _range];
+if (count _allBuildings == 0) then {
+	while {(_range < 1000) AND (count _allBuildings == 0)} do {
+		_range = _range + 100;
+		_allBuildings = nearestObjects [_targetPosition, ["Building"], _range];
+	};
+};
+
+if (count _allBuildings == 0) exitWith {};
+
+//@Stef using a random buidling or perhaps a special building?
+_targetBuilding = selectRandom _allBuildings;
+
+_duration = 30;
+_endTime = [date select 0, date select 1, date select 2, date select 3, (date select 4) + _duration];
+_endTime = dateToNumber _endTime;
+
+_task = ["SUP",[side_blue,civilian],[[_tskDesc,_targetName,numberToDate [2035,_endTime] select 3,numberToDate [2035,_endTime] select 4],_tskTitle,_marker],_targetPosition,"CREATED",5,true,true,"Heal"] call BIS_fnc_setTask;
+misiones pushBack _task;
+publicVariable "misiones";
+
+_spawnPosition = (getMarkerPos guer_respawn) findEmptyPosition [5,50,"C_Van_01_transport_F"];
```

Comment:
negative, vehicle should be brought by player

Reply:
Thanks, I was looking for something like this.
Vehicle will be gone the next push